src/documents/adding.md

```diff
@@ -97,99 +98,30 @@ Your flying car has an auto-pilot feature, and you want to add a preference for
-
-    ```html
-    <ul class="gpii-fd-navIcon-outer">
-        <li class="gpiic-fd-navIcon gpii-fd-navIcon gpii-fd-lang">
-            <div class="gpiic-fd-activeIndicator gpii-fd-activeIndicator"></div>
-            <div class="gpiic-fd-confirmedIndicator gpii-fd-confirmedIndicator"></div>
-        </li>
-        <li class="gpiic-fd-navIcon gpii-fd-navIcon-welcome">
-        </li>
-        <li class="gpiic-fd-navIcon gpii-fd-navIcon gpii-fd-showSounds">
-            <div class="gpiic-fd-activeIndicator gpii-fd-activeIndicator"></div>
-            <div class="gpiic-fd-confirmedIndicator gpii-fd-confirmedIndicator"></div>
-        </li>
-        <li class="gpiic-fd-navIcon gpii-fd-navIcon gpii-fd-autoPilot">
-            <div class="gpiic-fd-activeIndicator gpii-fd-activeIndicator"></div>
-            <div class="gpiic-fd-confirmedIndicator gpii-fd-confirmedIndicator"></div>
-        </li>
-        <li class="gpiic-fd-navIcon gpii-fd-navIcon gpii-fd-volume">
-            <div class="gpiic-fd-activeIndicator gpii-fd-activeIndicator"></div>
-            <div class="gpiic-fd-confirmedIndicator gpii-fd-confirmedIndicator"></div>
-        </li>
-           <!-- some list items not shown -->
-    </ul>
-    ```
-
-    Note that only the last class name on the `<li>` itself has changed.
-
-4. Edit `src/html/firstDiscovery.html` to add a placeholder for the panel.
-
-    A. Add a new `<section>` element of the same form as the others, but define the class names to reflect the auto-pilot preference:
+    A. Add a new `<section>` element of the same form as the others, but define the class names to reflect the "Autopilot" preference:
 
     ```html
     <section class="gpiic-fd-prefsEditor-panel-lang gpiic-fd-prefsEditor-panel gpii-fd-prefsEditor-panel-lang gpii-fd-main"></section>
     <section class="gpiic-fd-prefsEditor-panel-welcome gpiic-fd-prefsEditor-panel gpii-fd-prefsEditor-panel-welcome gpii-fd-main"></section>
     <section class="gpiic-fd-prefsEditor-panel-showSounds gpiic-fd-prefsEditor-panel gpii-fd-prefsEditor-panel-showSounds gpii-fd-main"></section>
-    <section class="gpiic-fd-prefsEditor-panel-autoPilot gpiic-fd-prefsEditor-panel gpii-fd-prefsEditor-panel-autoPilot gpii-fd-main"></section>
-    <section class="gpiic-fd-prefsEditor-panel-speakText gpiic-fd-prefsEditor-panel gpii-fd-prefsEditor-panel-speakText gpii-fd-main"></section>
-    <section class="gpiic-fd-prefsEditor-panel-speechRate gpiic-fd-prefsEditor-panel gpii-fd-prefsEditor-panel-speechRate gpii-fd-main"></section>
-    <section class="gpiic-fd-prefsEditor-panel-size gpiic-fd-prefsEditor-panel gpii-fd-prefsEditor-panel-size gpii-fd-main"></section>
-    <section class="gpiic-fd-prefsEditor-panel-keyboard gpiic-fd-prefsEditor-panel gpii-fd-prefsEditor-panel-keyboard gpii-fd-main"></section>
-    <section class="gpiic-fd-prefsEditor-panel-congratulations gpiic-fd-prefsEditor-panel gpii-fd-prefsEditor-panel-congratulations gpii-fd-main"></section>
```

Comment:
They are removed to conserve space. Its not necessary to keep all of them. There are only enough to get an idea of the pattern, and to recognize the correct location to insert the code. Having all of the preferences sections is confusing. Too much code to look at. It is noted in the code section that there is content that is not being shown there.